README.md

```diff
@@ -199,6 +198,30 @@ file `~/.oauth2l` by default.
 $ oauth2l reset
 ```
 
+### web
+
+Runs the web application and opens a new window to show it. If the web application is not installed it is installed in `~/.oauth2l-web` by default.
+
+```bash
+$ oauth2l web
+```
+
+## web --stop
```

Comment:
Move the documentation for "web --stop" and "web --directory" back to the command options section where you had it before : )

Reply:
Fixed!